cli/rt/02_console.js

```diff
@@ -469,6 +469,15 @@
       );
   }
 
+  // Surround a string with quotes when it is required (e.g the string not a valid identifier).
+  function maybeQuoteString(string) {
+    if (/^[a-zA-Z_][a-zA-Z_0-9]*$/.test(string)) {
```

Comment:
Technically we could be more permissive than this but this matches what Node does as far as I can tell.